guide/slash-commands/permissions.md

```diff
@@ -0,0 +1,74 @@
+# Slash command permissions
+
+Slash commands have their own permissions system. This system allows you to set the default level of permissions required for a user to to execute a command when its first deployed or your bot is added to a new server.
+
+The slash command permissions for guilds are defaults only and can be altered by guild administrators, allowing them to configure access however best suits their moderation and server roles. Your code should not try to enforce its own permission management, as this can result in a conflict between the server-configured permissions and your bot's code.
+
+::: warning
+It is not possible to prevent users with Administrator permissions from using any commands deployed to their guild. Think twice before creating "dev-only" commands such as an `eval`.
```

Comment:
```suggestion
It is not possible to prevent users with Administrator permissions from using any commands deployed globally or specifically to their guild. Think twice before creating "dev-only" commands such as `eval`.
```